network/src/router.rs

```diff
@@ -233,11 +271,31 @@ impl<P: ProvideRuntimeApi + Send, E, N, T> TableRouter for Router<P, E, N, T> wh
 		// give to network to make available.
 		self.fetcher.knowledge().lock().note_candidate(hash, Some(collation.pov), Some(extrinsic));
 		self.network().gossip_message(self.attestation_topic, statement.into());
+
+		for chunk in chunks.1 {
+			let signature = self.table.sign_chunk(chunk, chunks.0);
+			let message = ErasureChunkMessage {
+				chunk: chunk.clone(),
+				sender: chunks.0,
+				signature
+			};
+
+			self.network().gossip_message(self.attestation_topic, message.into());
+		}
 	}
 
 	fn fetch_pov_block(&self, candidate: &CandidateReceipt) -> Self::FetchValidationProof {
 		self.fetcher.fetch_pov_block(candidate)
 	}
+
+	fn fetch_erasure_chunk(
+		&self,
+		relay_parent: Hash,
```

Comment:
suggest changing the argument types here to type aliases instead, as mentioned here https://github.com/paritytech/polkadot/pull/345/files#r308998575